Code/Engine/Foundation/Algorithm/HashHelperString.h

```diff
@@ -10,9 +10,9 @@
 /// \brief Hash helper to be used as a template argument to ezHashTable / ezHashSet for case insensitive string keys.
 struct EZ_FOUNDATION_DLL ezHashHelperString_NoCase
 {
-  inline static ezUInt32 Hash(ezStringView szValue); // [tested]
+  inline static ezUInt32 Hash(ezStringView sValue); // [tested]
 
-  EZ_ALWAYS_INLINE static bool Equal(ezStringView lhs, ezStringView rhs); // [tested]
+  EZ_ALWAYS_INLINE static bool Equal(ezStringView sLhs, ezStringView sRhs); // [tested]
```

Comment:
Didn't you say **lhs** was allowed?

Reply:
No, I seid this is one of the points open for dsicussion ;-)

Reply:
I'd allow lhs and rhs as they only pop up in operators that always have the exact same pattern. You would never be confused about their type etc.

Reply:
Agreed